sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/interpreter/BeamSqlFnExecutorTestBase.java

```diff
@@ -83,7 +81,7 @@ public static void prepare() {
             .context(Contexts.EMPTY_CONTEXT)
             .ruleSets(BeamRuleSets.getRuleSets())
             .costFactory(null)
```

Comment:
@apilloud Could you run unit tests based on this class again? Some tests failed on my machine. It seems this test also need `CalciteConnection`.
Correct me if I am wrong

Reply:
Interesting. The tests pass for me. How are you running them? Are you using maven or gradle?

Reply:
@vectorijk Tests passed for me in both maven and gradle, and is passing in Jenkins. The issue you are seeing suggests that our JDBC driver is not being registered in your environment. This should be automatically done by the service file (`META-INF/services/java.sql.Driver`). Can you try manually loading the driver at the start of the test? This line should do it: 
```
Class.forName("org.apache.beam.sdk.extensions.sql.impl.JdbcDriver");
```

Reply:
Thanks! I think this is the issue. I will load the driver manually and test it again.